component-library/src/scss/_documentation-pages.scss

```diff
@@ -0,0 +1,242 @@
+@use '../../../core-library/util/device' as device;
+
+.content-doc-wrapper {
+  box-sizing: border-box;
+  max-width: 900px;
+  margin: 0 auto;
+
+  img {
+    width: 100%;
+  }
+}
+
+h3 {
```

Comment:
We shouldn't be redoing any styles. We should use the styles provided by the Design system.

Reply:
Fixed.